lib/prerender-lambda-construct.ts

```diff
@@ -1,16 +1,16 @@
-import { StackProps, Stack, Construct, CfnOutput } from '@aws-cdk/core';
+import { StackProps, Construct, CfnOutput } from '@aws-cdk/core';
 import { PrerenderFunction } from './prerender-construct';
 import { PrerenderCheckFunction } from './prerender-check-construct';
 
-export interface PrerenderLambdaStackProps extends StackProps {
+export interface PrerenderLambdaStack extends StackProps {
```

Comment:
Probably should rename this `PrerenderLambdaProps` and drop the extension of StackProps

Reply:
Have updated. I am going to merge as I need this available for some client work.